packages/toolpad-utils/src/check.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
I would not create separate utilities for these checks, I think we can remove them and their tests until we see that we need to reuse this type of logic in many different places.
We can do the checks inline for now, as needed.